nativeauthenticator/handlers.py

```diff
@@ -400,24 +482,27 @@ async def post(self):
             self.redirect(self.get_next_url(user))
         else:
             # default error mesage on unsuccessful login
-            error = "Invalid username or password"
+            error = "Invalid username or password."
```

Comment:
Off topic bug to fix separately, just a quick mention:
I got this when I used an invalid 2FA code

Reply:
I have opened #185 to track this problem.